app/_includes/md/mesh/install-universal-quickstart.md

```diff
@@ -12,5 +12,5 @@ The Kuma quickstart documentation is fully compatible with
 binaries instead of the Kuma binaries.
 
 To start using {{site.mesh_product_name}}, see the
-[quickstart guide for Universal deployments](https://kuma.io/docs/latest/quickstart/universal/).
+[quickstart guide for Universal deployments](https://kuma.io/docs/1.8.x/quickstart/universal/).
```

Comment:
Should we hardcode the version here or should we use `{{ page.version }}`?

Reply:
so before Kuma 2.0.x Kong Mesh was versioned differently from Kuma so {{ page.version }} would be wrong. What's fine though is that only KM 1.9 inherits from Kuma so it's safe to do this. (Also 1.9 is out of support so I don't think this matters much)